app/views/sessions/new.html.haml

```diff
@@ -4,7 +4,7 @@
       $('#nick').focus();
     });
 
-- title t('.title')
+- title FoodsoftConfig[:name] || t('.title')
```

Comment:
I'm not really happy with this. I'd prefer to keep the `t('.title')` in some form on the login page and use something like `t('.title', FoodSoftConfig[:name])` to pass the name to the translation file, but this will probably result in (too) long lines. what you think about just adding something like `h2 FoodsoftConfig[:name]` after the `title`?

Reply:
Done. Indeed it looks better now with a subtitle.